src/main/java/org/jenkinsci/plugins/tuleap_git_branch_source/webhook/TuleapWebHook.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.plugins.tuleap_git_branch_source.webhook;
+
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import hudson.util.HttpResponses;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import javax.annotation.CheckForNull;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Extension
+public class TuleapWebHook implements UnprotectedRootAction {
+
+    private static final Logger LOGGER = Logger.getLogger(TuleapWebHook.class.getName());
+
+    static final String WEBHOOK_URL = "tuleap-hook";
```

Comment:
Should not this have a an explicit visibility modifier?

Reply:
If there is no visibility, it means this is only visible inside the package only.
For now I do not need that this attribute is visible everywhere in the plugin.

Reply:
Should not it be visible only by the class then?

Reply:
No because it is also used in the TuleapWebHookCrumbExclusion class.